app.js

```diff
@@ -40,6 +41,9 @@ app.post("/api/articles/:article_id/comments", postComment)
 
 app.patch("/api/articles/:article_id", patchArticle);
 
+app.delete("/api/comments/:comment_id", deleteComment);
+
+
 
```

Comment:
Where do you refer to your error handlers?

Reply:
Hi @nc-niamh-smith do you mean where is the file? There is a folder called errors with an index.js file with all error handler function in it.

Reply:
No I understand that - I'm unsure as to where you are referring to it following the MVC model? I'm sure it all works correctly, but usually you would have a reference inside an app.use, for example: app.use(errorHandler). I'm just wondering how your errors make it to the separate file?